src/index.ts

```diff
@@ -28,7 +29,9 @@ async function route(request: Request) {
 		if (typeof text === 'string') {
 			const args = text.split(" ");
 
-			if (args.length === 1 && !args[0]) {
+			if (args.length === 1 && (!args[0] || args[0] === "help") {
```

Comment:
```suggestion
			if (args.length === 1 && (!args[0] || args[0] === "help")) {
```
Don't commit from your phone 😛

Reply:
Akash is clutch